### ISSUE TITLE
Talkback doesn't read "Syncing" when user taps Sync now item

### ISSUE THREAD

**Issue by asadotzler** [mozilla-mobile/fenix#2460]
## Steps to reproduce
Open Fenix with a Firefox account and Talkback running
Go to Settings
Go to Account settings
Focus and activate "Sync now"

### Expected behavior
When the text changes from "Sync now" to "Syncing", Talkback should read the "Syncing"

### Actual behavior
Talkback doesn't read the updated text. 

### Device information
* Android device: Samsung Galaxy S10e
* Fenix version: Saturday 5/11 @ 6:06 AM

**Comment by eeejay** [comment]
I think the label needs to be set as a live region.

**Comment by ekager** [comment]
@eeejay we have a status observer for sync status that we get a call for sync started ("syncing"?), sync successful, and sync error. We could easily add an accessibility announcement in these spots? I would just need strings for the last two

**Comment by eeejay** [comment]
Does the user see any kind of label for these different statuses? If so, the same language should be used here. Before you add accessibility announcements, can you try a [live region](https://developer.android.com/reference/android/view/accessibility/AccessibilityNodeInfo.html#setLiveRegion(int)) on a label that is already being used? This gives accessibility services a bit more context and the user some flexibility about how these things are presented.

If that won't work, an announcement will do.

**Comment by ekager** [comment]
No, the user sees the label change from "Sync Now", to "Syncing", and then "Sync Now" no matter the outcome, but maybe more context to what's happening could be helpful for all users...

Because it's a preference I'm having trouble getting the live region working because it's not a traditional view

**Comment by ekager** [comment]
Ah looks like there is a ticket to improve error states
https://github.com/mozilla-mobile/fenix/issues/2327

**Comment by bsurd** [comment]
Talkback does not announce the "Last synced: # minutes ago" if you don't let it finish reading the initial text (Sync now/Last..).

It just reads the Syncing part. Reproduced this in the latest build (5/15) on a OnePlus 5T and Nexus 5.

**Comment by ekager** [comment]
Thanks @bsurd , I'm going to close this for now because this screen will be refined for MVP see https://github.com/mozilla-mobile/fenix/issues/2325
